schema.graphql

```diff
@@ -0,0 +1,122 @@
+# Date without time, serialized as a YYYY-MM-DD string
+scalar DateOnly
+
+# Date and time, serialized as an ISO date string
+scalar DateTime
+
+# A valid email address
+scalar Email
+
+# A full URL with the protocol. Accepted protocols are http and https only.
+scalar Url
+
+#A notable person's label, e.g. Liberal, Democrat
+type Label {
+  id: ID!
+  text: String!
+  createdAt: DateTime!
+}
+
+# A comment on a notable person's event
+type NotablePersonEventComment {
+  owner: User!
+  text: String!
+}
+
+# An event about a notable person, must have a source link
+type NotablePersonEvent {
```

Comment:
We could add an `enum` type to the schema and a corresponding `Enum` type to the entity. I don't think it would break anything because we are just _adding_ a new property.